packages/core/types/src/modules/documents/params/index.ts

```diff
@@ -43,7 +43,7 @@ export type Pick<TSchemaUID extends UID.Schema, TKind extends Kind> = MatchAllIn
     // Publication Status
     [HasMember<TKind, 'status'>, PublicationStatus.Param],
     // Locale
-    [HasMember<TKind, 'locale'>, { locale?: Locale }], // TODO: also allow arrays ?
+    [HasMember<TKind, 'locale'>, { locale?: Locale | Locale[] }],
```

Comment:
We could have followed the same approach as for the rest of the params with different notation (Locale.StringNotation, Locale.ArrayNotation and Locale.Any) so that methods can decide what format they accept. What do you think?